hack/test-integration.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-ENVTEST_K8S_VERSION=${ENVTEST_K8S_VERSION:-"1.25"}
+ENVTEST_K8S_VERSION=${ENVTEST_K8S_VERSION:-"1.26"}
```

Comment:
There is also
- https://github.com/gardener/gardener/blob/c1290260a36853253cb0aef10220f30968e036f6/test/integration/resourcemanager/highavailabilityconfig/highavailabilityconfig_suite_test.go#L105-L106
- https://github.com/gardener/gardener/blob/c1290260a36853253cb0aef10220f30968e036f6/test/integration/resourcemanager/garbagecollector/garbagecollector_suite_test.go#L113-L114